### PR TITLE
Readme.md update Mac building from Windows. 

### PR DESCRIPTION
**Pull request by qwinci**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,27 @@ Your goal is to produce shapes by cutting, rotating, merging and painting parts
 
 **Notice**: This will produce a debug build with several debugging flags enabled. If you want to disable them, modify `config.js`.
 
+-   Mac building with Windows
```
Comment (tobspr) [inline README.md:25]: Could you make that a heading?

File: README.md [hunk 2]
```diff
@@ -22,6 +22,27 @@ Your goal is to produce shapes by cutting, rotating, merging and painting parts
...
+-   Make sure git `git lfs` extension is on your path
+-   Run `git lfs pull` to download sound assets
+-   Make sure `ffmpeg` is on your path
+-   Install Node.js and Yarn
+-   Be sure to run cmd as ADMIN! Otherwise this will not work.
+-   Run yarn in the root folder, then run 'yarn' in the `gulp/` folder
+-   Cd into `gulp` and run `yarn gulp main.standalone`
+-   Go into 'tmp_standalone_files' folder with file explorer and navigate into shapez.io-standalone-darwin-x64 or x32 folder
+-   Navigate into shapez.io-standalone.app/Contents/Frameworks folder
```
Comment (tobspr) [inline README.md:34]: I think everything before this line can be dropped since its already in the regular readme

File: README.md [hunk 3]
```diff
@@ -22,6 +22,27 @@ Your goal is to produce shapes by cutting, rotating, merging and painting parts
...
+-   This needs to be done after every rebuild or update!
+-   Do this for every .framework folder
+-   Step 1: Delete everything here except 'Versions' folder.
+-   Step 2: Navigate into .framework/Versions/A folder
+-   Step 3: Copy everything from there into .framework root folder
+-   To run game on Mac, run play.sh or just run Stapez.io.App
+
+**Notice**: This will produce a standalone build.
+
+-   If u get error when running 'yarn' in the `gulp/`, run 'yarn upgrade' in the `gulp/` folder and try again.
```
Comment (tobspr) [inline README.md:44]: No, thats a bad idea.

File: README.md (hunk 1 above)
Comment (qwinci) [reply README.md:25]: How?

File: README.md (hunk 2 above)
Comment (qwinci) [reply README.md:34]: Maybe, but then i need to exclude run cmd as admin in that section, otherwise it wont work.

File: README.md (hunk 3 above)
Comment (qwinci) [reply README.md:44]: Some node gyp error. Atleast now it works. I can remove it from readme if u want.

File: README.md [hunk 4]
```diff
@@ -22,6 +22,21 @@ Your goal is to produce shapes by cutting, rotating, merging and painting parts
 
 **Notice**: This will produce a debug build with several debugging flags enabled. If you want to disable them, modify `config.js`.
 
+## Mac building with Windows
+-   Be sure to run cmd as ADMIN! Otherwise this will not work.
```
Comment (tobspr) [inline README.md:26]: Don't use `ADMIN!` here that doesn't fit the style of the readme

File: README.md (hunk 2 above)
Comment (tobspr) [reply README.md:30]: Well then just write `Execute the regular build steps as admin`

File: README.md [hunk 5]
```diff
@@ -22,6 +22,19 @@ Your goal is to produce shapes by cutting, rotating, merging and painting parts
...
+## Mac building with Windows
+-   Execute the regular build steps as admin
+-   Go into 'tmp_standalone_files' folder with file explorer and navigate into shapez.io-standalone-darwin-x64 or x32 folder
+-   Navigate into shapez.io-standalone.app/Contents/Frameworks folder
+-   This needs to be done after every rebuild or update
+-   Do this for every .framework folder
+-   Step 1: Delete everything here except 'Versions' folder.
+-   Step 2: Navigate into .framework/Versions/A folder
+-   Step 3: Copy everything from there into .framework root folder
+-   To run game on Mac, run play.sh or just run Stapez.io.App
```
Comment (tobspr) [inline README.md:34]: There is a typo here

**Comment by tobspr** [comment]
Closed because I updated the build scrpit to manually do this :)

**Pull request closed without merging**